# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1313410270 -7200
#      Mon Aug 15 14:11:10 2011 +0200
# Node ID ccd17157dabf155a8a3c9e186242654df7b1c21c
# Parent  269ef479ecbf49e26580eafc03be012ba5b1ae19
IRC client alerts now when a personal message arrives.

diff --git a/gui/src/IRCWidget.cpp b/gui/src/IRCWidget.cpp
--- a/gui/src/IRCWidget.cpp
+++ b/gui/src/IRCWidget.cpp
@@ -21,16 +21,17 @@
 #include <QMessageBox>
 #include <QHBoxLayout>
 #include <QVBoxLayout>
 #include <QLabel>
 #include <QSettings>
 #include <QInputDialog>
 #include <QKeyEvent>
 #include <QScrollBar>
+#include <QApplication>
 #include "IRCClientImpl.h"
 
 ChatMessageTextEdit::ChatMessageTextEdit (QWidget *parent)
   : QTextEdit (parent), m_completer (0)
 {
   setMaximumHeight (50);
   setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Minimum);
 }
@@ -257,16 +258,18 @@ IRCWidget::showMessage (const QString& c
 {
   Q_UNUSED (channel);
   QString output;
   if (message.contains (m_ircClientInterface->nickname ()))
     {
       output =
         QString ("<font color=\"#990000\"><b>%1:</b> %2</font>").arg (sender).
         arg (message);
+
+      QApplication::alert (this);
     }
   else
     {
       output =
         QString ("<b>%1:</b> %2").arg (sender).
         arg (message);
     }
   m_chatWindow->append (output);
